### ISSUE TITLE
F-Droid inclusion

### ISSUE THREAD

**Issue by IzzySoft** [TrainerDex/AndroidApp#3]
Hi @TurnrDev, you've opened an [inclusion request](https://gitlab.com/fdroid/rfp/-/issues/1849) with us which is requiring your attention. I assume the notifications from GitLab have gone lost, so I decided to ask via an issue here. Quoting from there:

> Did you consider adding Fastlane to the app's repo? Be welcome to use my [Fastlane Cheat Sheet](https://gitlab.com/-/snippets/1895688) for a quick start. Placing `short_description.txt` with the (English) summary resp. `full_description.txt` for the full description to `/fastlane/metadata/android/en-US` would do for starters, though I'd highly recommend to provide some screenshots (and an icon) as well.

We'd welcome some feedback so we hopefully can process and complete your request with us :wink: Thanks in advance!

**Comment by TurnrDev** [comment]
Oops! Looks like that notification did get lost in a sea of emails! Thank you for checking in here. I've never heard of Fastlane but I'll check it out tonight or tomorrow night and comment on the request when I've implemented those changes :)

**Comment by IzzySoft** [comment]
Oh, I know that sea of mails… So all the best for setting up fastlane. If you want me to check the structure before you tag it, just ring ma bell!

**Comment by IzzySoft** [comment]
So any chance you get that going? For starters, `short_description.txt` and `full_description.txt` for `en-US` would do, though I'd strongly suggest to also include some screenshots for visuals.

I'll prolong the due-date on the MR once more now, but in a month it will enter the "candidate-to-close" phase if nothing happens…

**Comment by TurnrDev** [comment]
Damn! I forgot! 😢

**Comment by IzzySoft** [comment]
> Damn! I forgot!

:hankey: happens :speak_no_evil:

> Is this good?

Thanks a lot! Text files are fine. Icon & FeatureGraphic are a bit huge (icon should be max 512x512 AFAIK, and featureGraphic 1024x500) – and yes, you could make the screenshots twice or 3 times the current resolution (twice would be a good compromise between details visible and not-too-big).

As F-Droid works at tag level, we'll then need a tag covering that. You can tag now (and have the pics replaced with the next release), or replace the pics first (and tag tomorrow or next week). If you can give me a note (comment here or in the RFP) when the tag is there, that would be much appreciated – I then can "move on" the RFP at our end.

Note: There's currently a bug in fdroidserver making it impossible to *remove* screenshots. Work around is replacing them, so make sure to "reuse" the file names should you want to replace screenshots (and avoid replacing `*.jpg` by `*.png`, else "double trouble") – at least until the bug is fixed (which might take a while) :wink:

**Comment by IzzySoft** [comment]
@TurnrDev forgot again? :see_no_evil: I can lift the due-date a last time, giving you another 2 weeks – but then the RFP enters closure mode due to inactivity. From that point it will be 4 weeks until it's getting closed. Hopefully it won't come to that – and it's not that much missing AFAICS: fixing the graphics and tagging a new release. Can you do that? :crossed_fingers:

**Comment by TurnrDev** [comment]
I did forget! I'm at the computer now, so I'll do it now :) Than you for being so patient

**Comment by TurnrDev** [comment]
HI @IzzySoft I have the images :) when you say tag, do you mean git tag? What should the tag be named? Thank you

**Comment by IzzySoft** [comment]
Yeah! Images look great now (though I personally prefer screenshots without frames, as they add no value while stealing valuable screen estate on small phones) – but your frames are at least not making up 50% just for background color :see_no_evil:

Yes, `git tag`. And let me look what seems to fit you best here… Ah, I see. You currently use `versionName` as tag name, totally fine. So we have the problem that the current `versionName` is `0.2.0` – and that tag already exists. Let's see what options we have:

* where there code changes justifying a new release to be tagged? Then please increase `versionCode` e.g. to `4`, set `versionName` e.g. to `0.2.1` – and the corresponding tag would be `0.2.1`.
* no code changes (or just very minor ones)? Then we can start at F-Droid using the commit fixing fastlane, and set the update pattern for the tag naming we just discussed.

Once that's decided (and applied, in case of a new tag) we can continue with the RFP :tada:

**Comment by TurnrDev** [comment]
Let's go for option 2 :) The only code changes since that tag are very minor so F-Droid can start at the latest commit and use tags going forward  :D

**Comment by IzzySoft** [comment]
Great! I've updated the RFP accordingly and included a YAML to start with. If you want to, you can use that to open your own merge request against fdroiddata.

From this point, someone else needs to take over, as I usually do not handle the games sector. Wish you best progress!
